english/ironman/jarvis/jarvis.md

```diff
@@ -0,0 +1 @@
+Just A Rather Very Intelligent System, or J.A.R.V.I.S. is Tony Stark's artificially intelligent  computer. It is programmed to speak with a male voice in a British accent. J.A.R.V.I.S. is now merged with Vision [Source](http://marvel-movies.wikia.com/wiki/J.A.R.V.I.S.).
```

Comment:
This line is longer than 120 characters. Could you break it up into multiple lines? That way it will be easier to read the raw files or view diffs on the command line.  No need to add blank lines in between, just press enter somewhere in the middle of the line.